### ISSUE TITLE
Support "full debug mode" with metaschema-cli and Metaschema models at runtime

### ISSUE THREAD

**Issue by aj-stein-gsa** [GSA/oscal-js#11]
An an OSCAL developer who wants to do local and automated development and testing, I would like a variant of oscal-js that can perform "debug mode" that:

- Downloads and installs releases _or_ SNAPSHOT builds of `metaschema-cli`, _not_ `oscal-cli` to
- test new syntax additions and bug fixes to the metaschema-java project; and
- build the model and constraint files from particular commits for running integration tests and confirm features and bug fixes are implemented correctly

I am definitely excited to help with this effort, although it may be a separate repo and project (perhaps a metaschema-js). Let's discuss further and come up with a game plan if @wandmagic and/or other maintainers like this idea.

**Comment by wandmagic** [comment]
yes i like the idea of metaschema js i'll make a PR using a fork of this repo this week
